DeviceIdentification/DeviceIdentification.cpp

```diff
@@ -20,6 +20,18 @@
 #include "DeviceIdentification.h"
 #include "IdentityProvider.h"
 #include <interfaces/IConfiguration.h>
+#include "tracing/Logging.h"
+#include "UtilsJsonRpc.h"
+#include <com/com.h>
```

Comment:
Check if you need all these header files. I think ids.h is not needed. What about com.h, core.h?